### PR TITLE
Clerical robot getting more stuff

### PR DESCRIPTION
**Pull request by ben10083**
Gave Clerical module a Denied stamp **(WHY WAS THIS IT'S EMAGGED ITEM?!)** and a tape roll.

If it's emagged it gets a Chameleon stamp and a electrified arm for self defence.

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -675,7 +675,11 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/weapon/form_printer(src)
 	src.modules += new /obj/item/weapon/gripper/paperwork(src)
 	src.modules += new /obj/item/weapon/hand_labeler(src)
-	src.emag = new /obj/item/weapon/stamp/denied(src)
+	src.modules += new /obj/item/weapon/tape_roll //allows it to place flyers
+	src.modules += new /obj/item/weapon/stamp/denied(src) //why was this even a emagged item before smh
+	src.emag = new /obj/item/weapon/stamp/chameleon(src)
+	src.emag = new /obj/item/weapon/melee/baton/robot/arm(src) //self defence
```
Comment (Lohikar) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: You can only grant one item like this.

**Comment by ben10083** [comment]
Oof, is there a way to have it do both?

**Comment by Arrow768** [comment]
Not really, without some bigger changes

**Comment by LordFowl** [comment]
This is why we test our code.

**Comment by ben10083** [comment]
@Arrow768 Fixed it !rw

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 2]
```diff
@@ -675,7 +675,10 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/weapon/form_printer(src)
 	src.modules += new /obj/item/weapon/gripper/paperwork(src)
 	src.modules += new /obj/item/weapon/hand_labeler(src)
-	src.emag = new /obj/item/weapon/stamp/denied(src)
+	src.modules += new /obj/item/weapon/tape_roll //allows it to place flyers
```
Comment (skull132) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: Missing argument `(src)`

Comment (skull132) [reply code/modules/mob/living/silicon/robot/robot_modules.dm]: aaaa

Comment (skull132) [reply code/modules/mob/living/silicon/robot/robot_modules.dm]: this was supposed to be a request for changes.

**Comment by ben10083** [comment]
@skull132 fixed.

**Pull request merged**